### ISSUE TITLE
Trimming the input of whitespaces

### ISSUE THREAD

**Issue by mukkachaitanya** [prasadtalasila/autolabcli#111]
### Description

The inputs of the inquirer have not trimmed off the white spaces and are considered a part of the input.

### Steps to Reproduce

Login using the `autolabjs init` command for a user say `testuser`:
1. Login using username `testuser` ; logs in.
2. Login using username `testuser ` ; fails. 

**Expected behavior:** The login must go thorugh by trimming off the whitespaces

**Actual behavior:** Outputs `Invalid Username or Password.`

**Comment by prasadtalasila** [comment]
@mukkachaitanya Good catch, but this is not a high priority at this point. Please let this issue simmer a bit and focus on other high priority points from the refactoring page.
